tests/test_ant_legacy_bms.py

```diff
@@ -0,0 +1,247 @@
+"""Test the ANT implementation."""
+
+import asyncio
+from collections.abc import Buffer
+from typing import Final
+from uuid import UUID
+
+from bleak.backends.characteristic import BleakGATTCharacteristic
+import pytest
+
+from custom_components.bms_ble.plugins.ant_legacy_bms import BMS
+from custom_components.bms_ble.plugins.basebms import BMSsample
+
+from .bluetooth import generate_ble_device
+from .conftest import MockBleakClient
+
+BT_ADDRESS = "aa:bb:cc:a2:34:56"
```

Comment:
In a MAC the first 3 bytes are the OUI (Organizationally Unique Identifier) which could be used to identify the type of BMS. So please leave those and mask the device specific ones (last 3 bytes).